src/sonic-yang-models/doc/Configuration.md

```diff
@@ -1455,17 +1456,49 @@ name as object key and member list as attribute.
 }
 ```
 
+### Syslog Rate Limit
+
+Host side configuration:

```

Comment:
It might be useful to define the units on the parameters here, or provide a link to the rsyslog man page.